track/track.py

```diff
@@ -319,6 +332,35 @@ def __repr__(self):
     def __len__(self):
         return len(self.bounds_history)
 
+    def get_best_human_track(track_meta, tag_precedence, min_confidence=-1):
```

Comment:
This looks like it should have a `@classmethod` or `@staticmethod` decorator

Reply:
done.